applications/Talk_AwaydApp.md

```diff
@@ -0,0 +1,205 @@
+ 
+### Team Code Repos
+
+- 
+- https://github.com/TalkAway?tab=repositories
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- https://github.com/riocryptohub/riocryptohub; 
+- https://rashaabdulrazzak.github.io/my-profile/#/; 
+- 
+### Team LinkedIn Profiles (if available)
+
+- https://tr.linkedin.com/in/rasha-abdulrazzak; 
+- https://br.linkedin.com/in/douglas-miranda-; 
+- https://br.linkedin.com/in/fernanda-paix%C3%A3o-1483ba240
+- 
+## Development Status :open_book:
+
+If you've already started implementing your project or it is part of a larger repository, please provide a link and a description of the code here. In any case, please provide some documentation on the research and other work you have conducted before applying. This could be:
+
+https://medium.com/@nomadelabel/decentralized-education-what-decentralization-do-we-want-c58b619828d9
+
+ https://gov.near.org/t/approved-july-august-2022-cultural-event-to-launch-payment-in-crypto-currency-on-the-talk-away-platform/23351
+ 
+## Development Roadmap :nut_and_bolt:
+
+1 month: dapp building  | start of project presentations to communities | dapps working check
+
+### Overview
+
+- **Total Estimated Duration:**  2 months
+- **Full-Time Equivalent (FTE):**  30 days (Building the dapp) with 5 people full time. 60 days spreading and reach out people with 5 people full time. 
+- **Total Costs:** Requested amount in USD for the whole project 2 milestones (e.g. 14,000 USD). 
+
+### Milestone 1 Example — Basic functionality
+
+- **Estimated duration:** 1 month
+- **FTE:**  1,5
+- **Costs:** 7,000 USD
+
+> :exclamation: **The default deliverables 0a-0d below are mandatory for all milestones**, and deliverable 0e at least for the last one. 
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+|| 1. | Develop out Polkadot & Kusama core functionalities | - Polkadot & Kusama chain integration/support in wallet;<br/>- Polkadot & Kusama core functionalities ( send, receive, create/restore wallet and etc);<br/>- Complete Polkadot & Kusama transaction history with advanced information;
+ | 2. | Cross-chain swap/bridge between multiple tokens/chains | Cross-chain swap support for Polkadot & Kusama (Allowing users to directly onboard to Polkadot & Kusama from different chains “BTC, ETH, BSC and FIAT” within a single click) 
+ | 3. |  The smart contract will handle the following: Create new user | It will create a new user, either a free student or a certificate student. |Create new tutor | It will create a tutor with information about the courses, services he will provide, and the price for his lesson. | Create new resources | Login/logout from the wallet and chack the balance.
```

Comment:
Will you be using ink! for the contract(s)? Why not use an existing swap solution instead of creating your own, seeing as it's not the main focus of the app?

Reply:
No. We will not use ink! Yes, we have reason. We don't need the swap functionality in our app.